### DISCUSSION TITLE
document comment

### DISCUSSION THREAD

**Discussion by Silentdoer** [vlang/v#15265] [Ideas]
may be can use:
``` v
/**
 * desc
 * @param a desc
 * @return desc
 * @author xx
 * @since v0.1.1
 * @link xx.uu
 */
 fn xxx(a int) int {}
```
like java?

**Comment by Silentdoer** [comment]
up

**Comment by JalonSolov** [comment]
No.  Long discussion on Discord decided that doc comments should start with `///` while regular comments should start with `//` for single line comments, and be surrounded by `/*` and `*/ for multiline comments (as they do now).

There is no need for the useless separate `/**` and `*/` lines for docs.
